.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -2,17 +2,27 @@
 name: Bug report
 about: Report a bug in Linea
 title: ''
-labels: bug, alpha-user
+labels: alpha-user, bug
```

Comment:
we can remove alpha-user at this point now that it's no longer in private alpha.